### DISCUSSION TITLE
CSS doesn't load perfectly in first time visit on production (CPanel)

### DISCUSSION THREAD

**Discussion by hatamizq** [vercel/next.js#33711] [Help]
### What version of Next.js are you using?

12.0.7

### What version of Node.js are you using?

14.17.6

### What browser are you using?

Chrome, Safari

### What operating system are you using?

macOS

### How are you deploying your application?

next build && next export

### Describe the Bug

my css and image (larger size) not loaded in first time visit, that makes my page looks crashed. example :
![image](https://user-images.githubusercontent.com/49744607/150680716-3f71e90b-c6e0-4005-acdf-0bfb46942376.png)

but if I reload the page, it seems perfect
![image](https://user-images.githubusercontent.com/49744607/150680768-7a2a7c6c-48c7-4bed-b686-8a38ff5bdd31.png)

It happen all the time, any browser is the same, even the chrome's incognito.
how can I fix this? any suggestion? I've tried many ways but it seems doesn't work.

I'm using global and the module css.

fyi I'm using next export that rendered `out` folder. then I upload that to `public_html` folder on my cpanel.

here's my _app.js
```js
import '../styles/globals.css'
import 'bootstrap/dist/css/bootstrap.min.css';
import '../styles/navbar.module.css'
import '../styles/footer.module.css'
import '../styles/newsletter.module.css'
import '../styles/scroll-top.module.css'
import '../styles/home-banner.module.css'
import '../styles/latestproduct.module.css'
import '../styles/productcard.module.css'
import '../styles/socialmedia.module.css'
import '../styles/promo-banner.module.css'
import '../styles/promo-kategori.module.css'
import '../styles/login-modal.module.css'
import '../styles/register-modal.module.css'
import '../styles/productbanner.module.css'
import '../styles/product-main.module.css'
import '../styles/productdetail-banner.module.css'
import '../styles/productdetail-explore.module.css'
import '../styles/productdetail-main.module.css'
import '../styles/ordersummary.module.css'
import '../styles/orderprocessbanner.module.css'
import '../styles/voucherhanging.module.css'
import '../styles/cart.module.css'
import '../styles/video-modal.module.css'
import '../styles/loadingscreen.module.css'

import SimpleReactLightbox from 'simple-react-lightbox'

function MyApp({ Component, pageProps }) {
  return (
  <SimpleReactLightbox>
    <Component {...pageProps} />
  </SimpleReactLightbox>)
}

export default MyApp
```

also my _document.js
```js
import Document from 'next/document'
import { ServerStyleSheet } from 'styled-components'

export default class MyDocument extends Document {
  static async getInitialProps(ctx) {
    const sheet = new ServerStyleSheet()
    const originalRenderPage = ctx.renderPage

    try {
      ctx.renderPage = () =>
        originalRenderPage({
          enhanceApp: (App) => (props) =>
            sheet.collectStyles(<App {...props} />),
        })

      const initialProps = await Document.getInitialProps(ctx)
      return {
        ...initialProps,
        styles: (
          <>
            {initialProps.styles}
            {sheet.getStyleElement()}
          </>
        ),
      }
    } finally {
      sheet.seal()
    }
  }
}
```

### Expected Behavior

both global and module css loaded perfectly so my page doesn't seems awful again 

### To Reproduce

simply go to https://petersonslab.id then u can see the CSS and larger image was not loaded successfully

**Comment by balazsorban44** [comment]
The page isn't available. Could you please create an isolated/minimal reproduction that I can verify to investigate this further? Try to exclude any code that isn't necessary for the supposed error to present itself. :pray:

Also, note that images above the fold that should load immediately need to add the `priority` prop https://nextjs.org/docs/api-reference/next/image#priority

**Comment by hatamizq** [comment]
thanks for the notes! the page is available now. there's a problem with the domain back then.

**Comment by balazsorban44** [comment]
This seems to be an issue in the way you are deploying the site. There are broken images when I try to load the page.

Make sure you also read, since you cannot use the `next/image` component the same way when you use `next export`:
https://nextjs.org/docs/api-reference/next/image#built-in-loaders
